docs/changelogs/CHANGES-pre-1.0.md

```diff
@@ -3051,51 +3051,51 @@ Also see v0.9.4-rc1 changelog, which has been amalgamated into this release.
+-   `--generate-config` now no longer requires a `--server-name` parameter when used on existing configuration files.. ([\#220](https://github.com/matrix-org/synapse/issues/220))
+-   Add `--print-pidfile` flag that controls the printing of the pid to stdout of the demonised process.. ([\#213](https://github.com/matrix-org/synapse/issues/213))
 
 Media Repository:
 
--   Fix bug where we picked a lower resolution image than requested. (PR #205)
--   Add support for specifying if a the media repository should dynamically thumbnail images or not. (PR #206)
+-   Fix bug where we picked a lower resolution image than requested.. ([\#205](https://github.com/matrix-org/synapse/issues/205))
+-   Add support for specifying if a the media repository should dynamically thumbnail images or not.. ([\#206](https://github.com/matrix-org/synapse/issues/206))
 
 Metrics:
 
--   Add statistics from the reactor to the metrics API. (PR #224, #225)
+-   Add statistics from the reactor to the metrics API.. ([\#224](https://github.com/matrix-org/synapse/issues/224), [\#225](https://github.com/matrix-org/synapse/issues/225))
 
 Demo Homeservers:
 
--   Fix starting the demo homeservers without rate-limiting enabled. (PR #182)
--   Fix enabling registration on demo homeservers (PR #223)
+-   Fix starting the demo homeservers without rate-limiting enabled.. ([\#182](https://github.com/matrix-org/synapse/issues/182))
+-   Fix enabling registration on demo homeservers. ([\#223](https://github.com/matrix-org/synapse/issues/223))
 
 Changes in synapse v0.9.4-rc1 (2015-07-21)
 ==========================================
 
 General:
 
 -   Add basic implementation of receipts. (SPEC-99)
--   Add support for configuration presets in room creation API. (PR #203)
+-   Add support for configuration presets in room creation API.. ([\#203](https://github.com/matrix-org/synapse/issues/203))
 -   Add auth event that limits the visibility of history for new users. (SPEC-134)
--   Add SAML2 login/registration support. (PR #201. Thanks Muthu Subramanian!)
--   Add client side key management APIs for end to end encryption. (PR #198)
+-   Add SAML2 login/registration support. Thanks Muthu Subramanian! ([\#201](https://github.com/matrix-org/synapse/issues/201))
+-   Add client side key management APIs for end to end encryption.. ([\#198](https://github.com/matrix-org/synapse/issues/198))
 -   Change power level semantics so that you cannot kick, ban or change power levels of users that have equal or greater power level than you. (SYN-192)
--   Improve performance by bulk inserting events where possible. (PR #193)
--   Improve performance by bulk verifying signatures where possible. (PR #194)
+-   Improve performance by bulk inserting events where possible.. ([\#193](https://github.com/matrix-org/synapse/issues/193))
+-   Improve performance by bulk verifying signatures where possible.. ([\#194](https://github.com/matrix-org/synapse/issues/194))
```

Comment:
Looks like we now have double full-stops in places.

Reply:
Easy enough to fix, there was a couple that were `!.` too. That's what I get for using regexes...